columnphysics/icepack_atmo.F90

```diff
@@ -904,8 +896,28 @@ subroutine icepack_atm_boundary(sfctype,                   &
       real (kind=dbl_kind) :: &
          l_uvel, l_vvel, l_Uref
 
+      logical (kind=log_kind), save :: &
+         first_call = .true.   ! first call flag
+
       character(len=*),parameter :: subname='(icepack_atm_boundary)'
 
+      !------------------------------------------------------------
+      ! Check optional arguments
+      ! This is used for ocean and ice sfctype, need to be careful
+      !------------------------------------------------------------
```

Comment:
I'm not sure I understand this comment, why does it say 

> This is used for ocean and ice sfctype

but then we just check `sfctype == 'ice'` ? What is "This" referring to?

Reply:
I need to clean up this comment.